languages/fr/tl_news_category.php

```diff
@@ -17,12 +17,16 @@
 /**
  * Fields
  */
-$GLOBALS['TL_LANG']['tl_news_category']['title']         = array('Titre', 'Veuillez, s\'il vous plaît, saisir un titre pour cette catégorie.');
-$GLOBALS['TL_LANG']['tl_news_category']['frontendTitle'] = array('Titre front office', 'Ici, vous pouvez saisir un titre de catégorie qui sera affiché en front office.');
-$GLOBALS['TL_LANG']['tl_news_category']['alias']         = array('Alias', 'L\'alias de la catégorie est une référence unique qui peut être utilisé à la place de son ID numérique.');
-$GLOBALS['TL_LANG']['tl_news_category']['cssClass']      = array('Classe CSS', 'Ici, vous pouvez saisir une classe CSS qui sera ajoutée à la catégorie dans le front office');
-$GLOBALS['TL_LANG']['tl_news_category']['jumpTo']        = array('Page de redirection', 'Ici, vous pouvez choisir la page à laquelle les visiteurs seront redirigés lorsque vous cliquez sur un lien de catégorie dans le modèle d\'actualité.');
-$GLOBALS['TL_LANG']['tl_news_category']['published']     = array('Publier la catégorie', 'Rendre la catégorie visible sur le site internet.');
+$GLOBALS['TL_LANG']['tl_news_category']['title']                      = ['Titre', 'Veuillez, s\'il vous plaît, saisir un titre pour cette catégorie.'];
+$GLOBALS['TL_LANG']['tl_news_category']['frontendTitle']              = ['Titre front office', 'Ici, vous pouvez saisir un titre de catégorie qui sera affiché en front office.'];
+$GLOBALS['TL_LANG']['tl_news_category']['alias']                      = ['Alias', 'L\'alias de la catégorie est une référence unique qui peut être utilisé à la place de son ID numérique.'];
+$GLOBALS['TL_LANG']['tl_news_category']['cssClass']                   = ['Classe CSS', 'Ici, vous pouvez saisir une classe CSS qui sera ajoutée à la catégorie dans le front office'];
+$GLOBALS['TL_LANG']['tl_news_category']['jumpTo']                     = ['Page de redirection', 'Ici, vous pouvez choisir la page à laquelle les visiteurs seront redirigés lorsque vous cliquez sur un lien de catégorie dans le modèle d\'actualité.'];
+$GLOBALS['TL_LANG']['tl_news_category']['jumpToNews']                 = ['Redéfinir les paramètres pour les nouvelles dans cette catégorie', 'Configurez les redirections pour les nouvelles dans cette catégorie.'];
+$GLOBALS['TL_LANG']['tl_news_category']['news_category_news_archive'] = ['Archives des actualités', 'Sélectionnez une archive de nouvelles.'];
+$GLOBALS['TL_LANG']['tl_news_category']['news_category_jumpTo']       = ['Page de redirection de catégorie', 'Sélectionnez une page sur laquelle un visiteur sera redirigé lorsqu\'un lien de catégorie est cliqué dans le modèle de nouvelles.'];
```

Comment:
Thx for the French translation!

Could you use "actualité" instead of "nouvelle" to keep the same noun as the French translation of the Contao back end?

Reply:
Thanks, changed in fork.

Reply:
Thanks. Could you make the following changes:

`des actualité` => `des actualités`
`les actualité` => `les actualités`
`de actualité` => `d\'actualité`

Reply:
Thanks, did translation with google, so your help is really appreciated. :)